husky_extensions/itembox.js

```diff
@@ -32,6 +32,8 @@ define(function() {
             hideSelectedCounter: false,
             defaultDisplayOption: 'top',
             actionIcon: 'fa-plus-circle',
+            publishedStateName: 'publishedState',
+            publishedName: 'published',
```

Comment:
This way you make the `itembox` dependent on the published functionality. Shouldn't it be quite easy to move these changes to the concrete itembox implementations?